src/flux/task-step.coffee

```diff
@@ -85,6 +85,14 @@ TaskStepConfig =
 
       step.content.questions?[0].formats?.indexOf('free-response') > -1
 
+    canTryAnother: (id) ->
+      step = @_get(id)
+      step? and
+        (step.has_recovery and step.correct_answer_id isnt step.answer_id) and
+        not Task.TaskStore.isTaskPastDue(step.task_id) and
```

Comment:
instead of [module loading order in a one off way](https://github.com/openstax/tutor-js/pull/695/files#diff-74c51a44b8ba99c878804b91182ab691R8), could we pull `isTaskPastDue` into a helper/stateless function?